lib/flags.ml

```diff
@@ -189,6 +189,11 @@ let print_hyps_limit = ref (None : int option)
 let set_print_hyps_limit n = print_hyps_limit := n
 let print_hyps_limit () = !print_hyps_limit
 
+(* Flag for compact display of goals *)
+
+let compact_context = ref false
+let is_compact_context() = !compact_context
```

Comment:
Done and renamed into "Printing Compact Contexts".

Should I also move hyps limit there? Seems to be the same kind of option.

Actually I am not sure to understand why flags.ml should not contain all the options of coq. When I want to test the value of some option and don't remember its name I always have to look for it for a while. Is it in flags? vernacentries? Is it in the file where the option is actually used? If yes, what is this file already? Etc.

The option mechanism allows for all this to be spead everywhere but, except for plugins of course, I am not sure this is a good practice.